rs-react-app/src/main.tsx

```diff
@@ -0,0 +1,28 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import { RouterProvider } from 'react-router-dom';
+import { router } from './router/router';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+const container = document.getElementById('root');
+
+if (!container) {
+  throw new Error('Root element not found');
+}
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 5 * 60 * 1000,
```

Comment:
I see 5 * 60 * 1000 several times - it's nice to create a constant with this value.